docs/cncf/self-assessment.md

```diff
@@ -0,0 +1,281 @@
+# Buildah Self-assessment
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+|||| | \-- | \-- | | Assessment Stage | Incomplete | | Software | [https://github.com/containers/buildah](https://github.com/containers/buildah) | | Security Provider | No | | Languages | Go | | SBOM | [https://github.com/containers/buildah/blob/main/go.mod](https://github.com/containers/buildah/blob/main/go.mod) |
+
+### Security links
+
+| Doc | url |
+| :---- | :---- |
+| Security file | [https://github.com/containers/buildah/blob/main/SECURITY.md](https://github.com/containers/buildah/blob/main/SECURITY.md) |
+
+## Overview
+
+Buildah is a tool that facilitates building Open Container Initiative (OCI) images. It provides a flexible and efficient way to create container images without requiring a running container daemon, emphasizing security through rootless builds and fine-grained control over the image creation process.
+
+### Background
+
+Buildah is a command-line tool designed for building OCI-compliant container images. Unlike traditional container build tools that require a daemon, Buildah operates directly as a command-line application, providing better security and flexibility.
+
+Key characteristics:
+
+- **Daemonless**: No background daemon process required for building images
+- **Rootless builds**: Supports building images without root privileges
+- **OCI-compliant**: Fully compliant with OCI image specifications
+- **Dockerfile compatibility**: Can build images from Dockerfiles/Containerfiles
+- **Fine-grained control**: Provides detailed control over the build process
+- **Integration**: Works seamlessly with Podman, Skopeo, and other container tools
+
+Buildah is part of the containers ecosystem and integrates with other tools like Podman, Skopeo, and CRI-O.
+
+### Actors
+
+* **Buildah CLI**: The main command-line interface that users interact with for building container images.
+
+* **Build context**: The filesystem context containing source code and build instructions.
```

Comment:
This point is not very clear to me, I wonder if build context should be part of `actors` or not. @nalind WDYT

Reply:
removed in latest. PTAL.